src/main/java/net/cytonic/cytosis/Cytosis.java

```diff
@@ -33,7 +32,7 @@ public class Cytosis {
     private static CommandManager COMMAND_MANAGER;
     private static CommandHandler COMMAND_HANDLER;
     private static FileManager FILE_MANAGER;
-
+    private static Manager manager;
```

Comment:
Why is it called Manager? Name it like anything that has to do with database + manager